rai_test_utils/rai_test_utils/datasets/tabular/tabular_data_utils.py

```diff
@@ -61,7 +61,7 @@ def create_adult_census_data(string_labels=False):
     return X_train, X_test, y_train, y_test, categorical_features
 
 
-def create_cancer_data():
+def create_cancer_data(return_dataframe=False):
```

Comment:
not sure if you care enough to change it, but sklearn would call this `as_frame`.
https://scikit-learn.org/stable/modules/generated/sklearn.datasets.fetch_openml.html#sklearn.datasets.fetch_openml

Reply:
Maybe we can do change this later.